instantout/actions.go

```diff
@@ -96,6 +100,15 @@ func (f *FSM) InitInstantOutAction(eventCtx fsm.EventContext) fsm.EventType {
 		reservationAmt += uint64(res.Value)
 		reservationIds = append(reservationIds, resId[:])
 		reservations = append(reservations, res)
+
+		// Check that the reservation expiry is larger than the cltv
+		// expiry of the swap, with an additional delta to allow for
+		// preimage reveal.
+		if int32(res.Expiry) < initCtx.cltvExpiry+htlcExpiryDelta {
+			return f.HandleError(fmt.Errorf("reservation %x has "+
+				"expiry %v which is less than the swap expiry %v",
```

Comment:
nit: maybe mention the delta here so it's not confusing if someone gets this error.